pyproject.toml

```
@@ -13,6 +13,15 @@ packages = [
 
 [tool.poetry.dependencies]
 python = ">=3.10, <3.13"
+boto3 = "*"
+urllib3 = "<2"
+pandas = "*"
+pyarrow = "^15.0.0"
+fastparquet = "^2024.2.0"
+currencyconverter = "^0.17.16"
```

```suggestion
currencyconverter = "^0.17.16"
```
is there specific cases that fastparquet is needed? the newer updates on pandas would be leaning to pyarrow being default.

---

Just wondering if you've encountered any errors @beingkk . As the parquet raw crunchbase outputs are using pyarrow

---

Hey - I think I probably had an error that suggested installing fastparquet, but I don't have strong feelings about this. Happy to use another approach if that makes more sense!